src/HonzaBotner/HonzaBotner.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
+    <Version>1.1.0</Version>
```

Comment:
Why add version only to this assembly and not all of them?

Reply:
Look at file Directory.Build.props. Content of this file is "injected" to all *.*proj files processed by MSBuild (dotnet build) during build.